addons/traefik-forward-auth/traefik-forward-auth.yaml

```diff
@@ -93,3 +95,7 @@ spec:
           value: "kubeaddons"
         - name: "TFA_INGRESS_SERVICE_NAME"
           value: "traefik-kubeaddons"
+      clusterStorage:
```

Comment:
I double checked and these additional changes are already in the master/other backport PRs 👍🏻